README.md

```diff
@@ -106,8 +117,11 @@ our company and culture evolve, so should this document.
 
 ### Invent
 
-#### Don't be afraid to push the boundaries of technology, but [be conservative with your tech stack](http://mcfunley.com/choose-boring-technology).
-- What this means is that the things you are building should be pushing boundaries, not the things you are building them with. [Here is a list of technology decisions we have evaluated in the past](https://mobify.atlassian.net/wiki/questions/77168927/what-should-i-consider-when-evaluating-a-new-service-feature-or-technology-what-criteria-do-we-use-in-our-decisions).
+#### Push the boundaries of technology, but [try to map new problems to our stack](http://mcfunley.com/choose-boring-technology).
+
+We're inventing a new product category. This is pretty tough.
+
+Favour battle-tested technologies that we're familiar with.
```

Comment:
Should we add something to the effect of "Choose old tech for new problems and consider new tech for 1-to-n solutions."   Also, the "Favour battle-tested" line goes with the previous line, doesn't it?... why the extra line break... makes them feel unrelated. :)